docs/UsersGuide.asciidoc

```diff
@@ -362,10 +362,10 @@ is `label:force_result:<new_result>[:<description>]`, for example
 this command to be effective the according user needs to have at least
 operator permissions.
 
-NOTE: `force_result`-labels are not evaluated when when a comment is
-<<UsersGuide.asciidoc#carry-over,carried over>>. The carry over will only
-happen when the comment *also* contains a bug reference which is therefore not
-recommended.
+NOTE: `force_result`-labels are evaluated when when a comment is
+<<UsersGuide.asciidoc#carry-over,carried over>>. However, the carry over will
+only happen when the comment *also* contains a bug reference which is therefore
+not recommended.
```

Comment:
Hm, I'm not sure how I should read that as a user.
so this feature is generally not recommended? But why?

Reply:
I've dropped the "not recommended" part as it is now actually not the case anymore.